build_windows/vs2013/cobc.vcxproj.filters

```diff
@@ -92,6 +95,9 @@
     <None Include="..\..\cobc\warning.def">
       <Filter>Header Files</Filter>
     </None>
+    <None Include="..\..\cobc\dump_ast_gen.c">
+      <Filter>Header Files</Filter>
+    </None>

```

Comment:
I'm confused - that is an auto-generated source (like parser.c) and therefore should be moved there and `ClCompile`d as well, no [that applies to all of those files under build_windows]?

Reply:
I don't really understand these files, I just grep for other similar files (warning.def and typeck.c) and modified the files accordinly...